scripts/west_commands/runners/jlink.py

```diff
@@ -30,6 +30,8 @@
 DEFAULT_JLINK_RTT_PORT = 19021
 
 def is_ip(ip):
```

Comment:
could probably be more concise, but not sure which version I prefer anyway

```python
def is_ip(ip):
    try:
        return ip is not None and ipaddress.ip_address(ip.split(':')[0])
    except (ValueError, AttributeError):
        return False
```

Reply:
This would return an IP address instead of a boolean.